metrics-operator/controllers/common/providers/datadog/common.go

```diff
@@ -0,0 +1,43 @@
+package datadog
```

Comment:
can you also make a test file for this guy? It should be quite similar to [the dynatrace common](https://github.com/keptn/lifecycle-toolkit/blob/main/metrics-operator/controllers/common/providers/dynatrace/common_test.go)

Reply:
yes, I also thought to add more test-cases. Will add